apps/web/src/app/arrangementer/components/AttendanceCard/PunishmentBox.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Punishment } from "@dotkomonline/types"
+import { Button, Icon, Text } from "@dotkomonline/ui"
+
+export const PunishmentBox = ({ punishment }: { punishment: Punishment }) => {
```

Comment:
Lets be consistent with declaring components. Most other places in the codebase uses FC